surveys/micro/leadership-council-2025.md

```diff
@@ -0,0 +1,159 @@
+# Survey questions
+
+One of the responsibilities of the [leadership council](https://www.rust-lang.org/governance/teams/leadership-council),
+formed by [RFC 3392](https://rust-lang.github.io/rfcs/3392-leadership-council.html),
+is to solicit feedback on a yearly basis from the Project on how we are performing
+our duties.
+
+> Each year, the Council must solicit feedback on whether the Council is
+> serving its purpose effectively from all willing and able Project members and
+> openly discuss this feedback in a forum that allows and encourages active
+> participation from all Project members. To do so, the Council and other
+> Project members consult the high-level duties, expectations, and constraints
+> listed in this RFC and any subsequent revisions thereof to determine if the
+> Council is meeting its duties and obligations.
+
+This is a quick annual survey for Rust-lang project members to provide feedback
+on the Leadership Council. Your feedback is appreciated to help steer the
+Council activities. As always, if you have specific questions or feedback that
+you would like to provide, feel free to reach out to any Council member with
+questions.
+
+This survey is anonymous. The Council will go through all answers and release a
+summary after the survey is complete, similar to the one published
+[last year](https://blog.rust-lang.org/2024/08/26/council-survey/).
+
+## Leadership Council Effectiveness
+
+For each of the key [duties] outlined in RFC 3392 (quoted below), we'd like your
+input on how well the Council is performing against that duty.
+
+You may want to (re)read the last ~year of regular updates from the Council to help build context:
+
+* [2024 Q3](https://blog.rust-lang.org/inside-rust/2024/09/06/leadership-council-update/)
+* [2024 Q4](https://blog.rust-lang.org/inside-rust/2024/12/09/leadership-council-update/)
+* [2025 Q1](https://blog.rust-lang.org/inside-rust/2025/03/17/leadership-council-update/)
+* [2025 Q2](https://blog.rust-lang.org/inside-rust/2025/06/11/leadership-council-update/)
+
+See also our [spending/budget](https://github.com/rust-lang/leadership-council/blob/main/policies/spending/budget.md).
+
+[duties]: https://rust-lang.github.io/rfcs/3392-leadership-council.html#duties-expectations-and-constraints-on-the-council
+
+### Do you consider yourself a member of the Rust Project?
+
+Type: select one (required)
+
+- Yes
+- No
+
+### What teams/areas of the project do you regularly participate in?
+
+This question aims to provide us with feedback on areas of the project that we
+may not be addressing as well (or areas we're doing better in).
+
+Type: select one or more (optional)
+
+- Compiler
+- Dev Tools
+- Launching Pad
+- Library
+- Infrastructure
+- Moderation
+- Language
+
+### Tracking and delegating work
+
+> Identifying, prioritizing, and tracking work that goes undone due to lack of
```

Comment:
Is this supposed to be an internal comment or should this be shown in the survey?